app/controllers/catalog_controller.rb

```diff
@@ -105,7 +112,9 @@ def self.modified_field
     config.add_show_field solr_name("date_modified", :stored_searchable), label: "Date Modified"
     config.add_show_field solr_name("date_uploaded", :stored_searchable), label: "Date Uploaded"
     config.add_show_field solr_name("date_valid", :stored_searchable), label: "Date Valid"
+    config.add_show_field solr_name("hydrologic_unit_code", :stored_searchable), label: "Hydrologic Unit Code"
```

Comment:
Arent these showing up on the single result show field already? Via the presenter override.